mon-pix/app/templates/components/challenge-embed-simulator.hbs

```diff
@@ -1,20 +1,20 @@
-{{#if _hiddenSimulatorClass}}
+{{#if this_hiddenSimulatorClass}}
```

Comment:
`this._hiddenSimulatorClass` non ?

Reply:
C'est corrigé dans le dernier commit je crois